pkg/sqlmigration/018_drop_license_sites.go

```diff
@@ -0,0 +1,56 @@
+package sqlmigration
+
+import (
+	"context"
+
+	"github.com/SigNoz/signoz/pkg/factory"
+	"github.com/uptrace/bun"
+	"github.com/uptrace/bun/migrate"
+)
+
+type schemaCleanup struct{}
+
+func NewDropLicensesSitesFactory() factory.ProviderFactory[SQLMigration, Config] {
+	return factory.NewProviderFactory(factory.MustNewName("drop-sites-licenses"), func(ctx context.Context, ps factory.ProviderSettings, c Config) (SQLMigration, error) {
```

Comment:
```suggestion
	return factory.NewProviderFactory(factory.MustNewName("drop_sites_licenses"), func(ctx context.Context, ps factory.ProviderSettings, c Config) (SQLMigration, error) {
```

The factory name must equal the file name without the number prefix